src/DataStructures/Tensor/EagerMath/Magnitude.hpp

```diff
@@ -8,6 +8,7 @@
 
 #include <cstddef>
 
+#include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
 #include "DataStructures/Tensor/Tensor.hpp"
 #include "Utilities/MakeWithValue.hpp"
```

Comment:
Remove MakeWithValue and cstddef as they are not used here

Reply:
`/// Defines functions euclidean_magnitude and magnitude` should be changed as there is no longer a `euclidean_magnitude` function

Reply:
I would just remove the Doxygen description of the file.